src/MMLib.SwaggerForOcelot/Transformation/SwaggerJsonTransformer.cs

```diff
@@ -85,17 +85,19 @@ private string TransformOpenApi(JObject openApi, IEnumerable<ReRouteOptions> reR
                     (downstreamUrl.IsAbsoluteUri ? downstreamUrl.AbsolutePath : downstreamUrl.OriginalString)
                     .RemoveSlashFromEnd();
             }
-            
+
             var paths = openApi[OpenApiProperties.Paths];
             if (paths != null)
             {
                 RenameAndRemovePaths(reRoutes, paths, downstreamBasePath);
 
-                RemoveItems<JProperty>(
-                    openApi[OpenApiProperties.Components][OpenApiProperties.Schemas],
-                    paths,
-                    i => $"$..[?(@*.$ref == '#/{OpenApiProperties.Components}/{OpenApiProperties.Schemas}/{i.Name}')]",
-                    i => $"$..[?(@*.*.items.$ref == '#/{OpenApiProperties.Components}/{OpenApiProperties.Schemas}/{i.Name}')]");
+                var schemaToken = openApi[OpenApiProperties.Components][OpenApiProperties.Schemas];
+                if (schemaToken != null)
```

Comment:
Surround `if` block body to `{}` parentheses.

Reply:
Fixed